<launch>
    <arg name="ns" default="" />
    <group ns="$(arg ns)">
        <arg name="pipeline_config_file" value="$(find percept)/config/sim_pipeline_conf.yaml"/>
        <rosparam file="$(arg pipeline_config_file)" command="load"/>
        <node name="perception_node" pkg="percept" type="sim_pipeline.py" output="screen">
          <param name="ns" value="$(arg ns)"/>
        </node>
        <!--<node name="perception_rviz" pkg="rviz" type="rviz" args="-d $(find percept)/rviz/perception.rviz"/> -->
    </group>
</launch>

